src/combinatorics/derangements/basic.lean

```diff
@@ -24,105 +25,134 @@ We also define:
     sigma-type `Σ a : α, {f : perm α // only_possible_fixed_point f a}`.
 -/
 
-open equiv
-
-namespace derangements
-
-section definitions
+lemma function.mem_fixed_points_iff {α : Type*} {f : α → α} {x : α} :
```

Comment:
do these belong in dynamics.fixed_points?

Reply:
Yeah, most likely. They are very special cases of the very general lemmas Yury proved there, but I think they are  quite handy to have. The thing is that I don't know whether we want to use `is_fixed_pt` or `fixed_points`, although I have a preference for the later.